### DISCUSSION TITLE
Repository for PDF-Templates

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#16960] [Wishlist]
Hello,

I saw that YF comes "only" with 3 PDF templates for invoices.

I have looked around a lot the last days for finding the variables etc. for creating other templates e.g., for 'Sales order', 'Goods dispached note / Delivery notice' or 'Quotes' and I was wondering, why YF does not come with some more basic PDF-templates.

Maybe there is a way to offer a repo here, where people can contribute & upload there templates as a base for further customization?

Kind regards

**Comment by mariuszkrzaczkowski** [comment]
We have a repository for such add-ons: https://github.com/YetiForceCompany/YetiForceScripts
